packages/common/src/events/edit.ts

```diff
@@ -118,14 +130,28 @@ export async function updateHubEvent(
   return mapper.storeToEntity(model, {}) as IHubEvent;
 }
 
+/**
+ * @private
+ * Create an Event registration
+ * @param data
+ * @param requestOptions
+ * @returns Promise<void>
+ */
+export async function createHubEventRegistration(
```

Comment:
Minor, but the `await` on line 144 is unnecessary since this method doesn't need to wait for the promise to resolve. Given that, we can drop the `async` on line 140 & the `await` on line 144, and simply return the promise returned by `createRegistration`.